### PR TITLE
enhance inspect

### PR DESCRIPTION
**Pull request by fahedouch**
- enhance https://github.com/containerd/nerdctl/issues/736
-  fix duplicate inspect entries

Signed-off-by: fahed dorgaa <fahed.dorgaa@gmail.com>

### CODE REVIEW

File: cmd/nerdctl/container_inspect.go [hunk 1]
```diff
@@ -69,18 +69,13 @@ func containerInspectAction(cmd *cobra.Command, args []string) error {
...
-		n, err := walker.Walk(ctx, req)
-		if err != nil {
-			errs = append(errs, err)
-		} else if n == 0 {
-			errs = append(errs, fmt.Errorf("no such object: %s", req))
-		}
-	}
-	if len(errs) > 0 {
-		return fmt.Errorf("%d errors: %v", len(errs), errs)
+	n, err := walker.Walk(ctx, args[0])
```
Comment (AkihiroSuda) [inline cmd/nerdctl/container_inspect.go:72]: Why only args[0]

File: cmd/nerdctl/fmtutil.go [hunk 2]
```diff
@@ -75,7 +75,12 @@ func formatSlice(cmd *cobra.Command, x []interface{}) error {
 		for _, f := range x {
 			var b bytes.Buffer
 			if err := tmpl.Execute(&b, f); err != nil {
-				return err
+				if _, ok := err.(template.ExecError); ok {
+					// FallBack to Raw Format
```
Comment (AkihiroSuda) [inline cmd/nerdctl/fmtutil.go:79]: Why do we need this

Comment (fahedouch) [reply cmd/nerdctl/fmtutil.go:79]: We need this to encode struct with `tags` (RawFormat) and execute tmpl on the  RawFormat in the case where `--format` point to tag like  `Id` and not  `ID`

File: cmd/nerdctl/container_inspect.go (hunk 1 above)
Comment (fahedouch) [reply cmd/nerdctl/container_inspect.go]: I forgot that we can inspect an image or a container directly 😅

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**